apis/v1alpha1/ack-generate-metadata.yaml

```diff
@@ -0,0 +1,14 @@
+ack_generate_info:
+  build_date: "2021-06-22T19:42:30Z"
+  build_hash: 38e309d07369b1e33250473084f91db3894c4379
+  go_version: go1.16.4 darwin/amd64
+  version: v0.2.3
+api_directory_checksum: 650887de24f94277e21771990d1391198cce93c3
+api_version: v1alpha1
+aws_sdk_go_version: ""
```

Comment:
@A-Hilaly why is this empty, is there a bug here?

Reply:
It's not really a bug, it's hard to pass the used "aws-sdk-go" version if not precised with an ENV var

If you rerun the generation using `AWS_SDK_GO_VERSION=...` it will appear in the `ack-generate-metadata.yaml`

Reply:
I'm working on a fix that will involve using `gopkg.in/go-git` libraries and a big rework of the cmds/code-gen probably coming in `0.2.4`